common/src/db/query/filtering.rs

```diff
@@ -16,7 +16,22 @@ pub trait Filtering<T: EntityTrait> {
     where
         Self: Sized + QueryFilter + QueryOrder,
     {
-        search.query(self, &context.columns())
+        let Query { ref q, ref sort } = search;
```

Comment:
interesting... 
https://doc.rust-lang.org/std/keyword.ref.html

`Using the ref keyword, the value is only borrowed, not moved, making it available for use after the match statement:`

I already saw clippy asking to avoid match statements containing only one arm, there is a direct correlation with if and else then.